### ISSUE TITLE
Section divider width

### ISSUE THREAD

**Issue by uvjim** [thomasloven/lovelace-fold-entity-row#195]
My Home Assistant version: 2021.11.5

Fold-entity-row version (FROM BROWSER CONSOLE): 20.0.12

What I am doing:
Creating any fold-entity-row with a section heading results in a shorter than it should be divider line.
I have restarted HASS, forced a cache reload and tried different browsers (screenshot is from FF but also tried in Edge).

What I expected to happen:
To have a line that spanned the width of the card.

What happened instead:
I get a shorter than expected divider line.

`blue` arrows show the edges of the section divider line in the card.
`green` arrows show the blank edges of the shorter section divider from fold-entity-row

![image](https://user-images.githubusercontent.com/8431065/144307086-dfd6988a-0a97-4229-a7b9-28fa17ee7776.png)

**Minimal** steps to reproduce:

A slightly modified version of an example in the README.

```yaml
type: entities
entities:
  - type: custom:auto-entities
    filter:
      include:
        - domain: device_tracker
    card:
      type: custom:fold-entity-row
      head:
        type: section
        label: Automatically populated
  - type: section
```

Error messages from the browser console: No errors relating to fold-entity-row

---

**By putting an X in the boxes ([X]) below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [X] Have made sure I am using the latest version of the plugin.
- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.
- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by weelad** [comment]
I'm seeing this too.

**Comment by ildar170975** [comment]
A possible workaround:
```
  - type: entities
    title: fold-entity-row
    entities:
      - entity: sun.sun
      - type: custom:fold-entity-row
        head:
          type: section
          label: Section
        padding: 5
        open: true
        entities:
          - sun.sun
          - sun.sun
        card_mod:
          style: |
            div#head hui-section-row {
              overflow: visible !important;
            }
```
![изображение](https://user-images.githubusercontent.com/71872483/147521745-3b6cf61d-db8f-48ea-9609-3bf3691d52f9.png)

**Comment by ildar170975** [comment]
The problem is not observed on iOS app (iOS 15.x)
HA 2021.6

**Comment by uvjim** [comment]
A quick change to the code to make it apply to all folds in the entities card...

```yaml
type: entities
title: fold-entity-row
card_mod:
  style:
    '#states':
      fold-entity-row:
        $: | 
          div#head hui-section-row {
            overflow: visible !important;
          }
entities:
  - entity: sun.sun
  - type: custom:fold-entity-row
    head:
      type: section
      label: Section
    padding: 5
    open: true
    entities:
      - sun.sun
      - sun.sun
  - type: custom:fold-entity-row
    head:
      type: section
      label: Section 2
    padding: 5
    open: true
    entities:
      - sun.sun
      - sun.sun
  - type: section
    label: Section 3
  - entity: sun.sun
```

![image](https://user-images.githubusercontent.com/8431065/151013700-114f50ec-0b30-4adf-a09b-cf9b722d8810.png)

**Comment by ildar170975** [comment]
Seems to be fixed in 2.0.15

**Comment by uvjim** [comment]
Confirmed. Seems to be resolved now.
